soroban-env-common/src/raw_val.rs

```diff
@@ -681,8 +681,8 @@ impl Debug for RawVal {
         }
 
         match self.get_tag() {
-            Tag::U32Val => write!(f, "U32({})", self.get_major() as u32),
-            Tag::I32Val => write!(f, "I32({})", (self.get_major() as u32) as i32),
+            Tag::U32Val => write!(f, "U32({})", { self.get_major() }),
```

Comment:
Does clippy require the {}? At a glance that looks unnecessary.

Reply:
Doesn't look like it. I removed the braces, ran `cargo clippy --fix` again,  and clippy didn't add the braces back in.